docs/en/get_started.md

```diff
@@ -243,11 +315,6 @@ cv2.imwrite('output_detection.png', img)
 
 You can find more examples from [here](https://github.com/open-mmlab/mmdeploy/tree/master/demo/python).
 
-```{note}
-If you build MMDeploy from source, please add ${MMDEPLOY_DIR}/build/lib to the environment variable PYTHONPATH.
-Otherwise, you will run into an error like ’ModuleNotFoundError: No module named 'mmdeploy_python'
-```
-
 #### C API
```

Comment:
Maybe we can replace the C API example with a C++ one, which is simpler and less verbose.